src/yaxunit/test-model/model.ts

```
@@ -2,13 +2,13 @@ import { IEvent, Emitter, IPosition } from 'monaco-editor-core'
 
 import { TestDefinition, TestStatus } from './types'
 import { Method } from '@/common/codeModel'
-import { Report, TestCaseResult, TestSuiteResult } from './report'
+import { Error, Report, TestCaseResult, TestSuiteResult } from './report'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)